tests/webview/ui/test_sitemap.py

```diff
@@ -31,3 +34,47 @@ def test_sitemap_is_segmented_by_author(webview_base_url, selenium, author_usern
     url_region = sitemap.url_regions[0]
     content = url_region.open()
     assert author_name in content.content_header.authors
+
+
+@markers.webview
+@markers.smoke
+@markers.requires_varnish_routing
+@markers.nondestructive
+@markers.parametrize(
+    "author, uuid, be_present",
+    [  # case parameters for the removal of derived content
+        (
+            "cnxbio_espanol",
+            # Conceptos de Biología
+            "e7a016d3-91fc-4ba0-9e05-a33e986f3d94",
+            False,
+        ),
+        # case parameters for keeping OpenStax derived content (i.e. REX book)
+        (
+            "OpenStaxCollege",
+            # Introduction to Sociology 2e
+            "02040312-72c8-441e-a685-20e9333f3e1d",
+            True,
+        ),
+    ],
+)
+def test_derived_copies_in_sitemap(webview_base_url, author, uuid, be_present):
+    # As OpenStax, I want to remove all derived copies except OpenStax
+    # books from the content, so that we begin the transition away from cnx.
+    # see: https://github.com/openstax/cnx/issues/727
+
+    # GIVEN a "sitemap" URL
+    # note, this is discovered by the spider as part of /sitemap_index.xml
+    url = f"{webview_base_url}/sitemap-{author}.xml"
+
+    # WHEN a spider requests the specific author's sitemap
+    response = requests.get(url)
+    response.raise_for_status()
+    xml = lxml.etree.fromstring(response.content)
+    #: give the text for the the element named 'loc' in any namespace
+    urls = xml.xpath("//*[local-name()='loc']/text()")
+
+    # THEN non-OpenStax derived copies have been removed
+    # while OpenStax books remain
+    presents_operation = be_present and (lambda x: x) or (lambda x: not x)
```

Comment:
I'd break this out a bit more for readability, either a conditional or named function. It definitely shows advanced usage of python features like lambda but it does so at the expense of readability. Upon first looking at the code I had to scratch my head for a moment, mostly in thinking about what `be_present` meant. This is a common thing with lambdas in my experience; they are particularly prone to poor naming. Additionally, I'm concerned about Otto maintaining this